### ISSUE TITLE
A method with no parameter gets a wrong error message saying `argument` instead of `parameter`

### ISSUE THREAD

**Issue by hyperkai** [python/mypy#20939]
*Notes:
- mypy test.py
- mypy 1.19.1
- Python 3.14.0

A method with no parameter gets the wrong error message saying `argument` instead of `parameter` as shown below:

```python
class Cls:
    def meth(): ...
# Error
```

> error: Method must have at least one argument. Did you forget the "self" argument?

So, the error message should say `parameter` as shown below:

> error: Method must have at least one parameter. Did you forget the "self" parameter?

**Comment by ilevkivskyi** [comment]
I don't think this has much value.
